colab/edge_detection.ipynb

```diff
@@ -16,18 +16,26 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "license"
+        "id": "FH3IRpYTta2v"
       },
       "source": [
-        "```\n",
-        "Copyright 2020 The IREE Authors\n",
-        "\n",
-        "Licensed under the Apache License v2.0 with LLVM Exceptions.\n",
-        "See https://llvm.org/LICENSE.txt for license information.\n",
-        "SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception\n",
-        "```\n"
+        "##### Copyright 2020 The IREE Authors"
       ]
     },
+    {
+      "cell_type": "code",
```

Comment:
ehh this isn't actually collapsed when I saved it...?

preview: https://colab.research.google.com/github/scotttodd/iree/blob/colab-refresh/colab/edge_detection.ipynb

(still better than the pure source block, but not quite what we'd like)

Reply:
(╯°□°)╯︵ ┻━┻

edited the .ipynb files directly to collapse them

previews:
* https://colab.research.google.com/github/scotttodd/iree/blob/colab-refresh/colab/edge_detection.ipynb
* https://colab.research.google.com/github/scotttodd/iree/blob/colab-refresh/colab/low_level_invoke_function.ipynb
* https://colab.research.google.com/github/scotttodd/iree/blob/colab-refresh/colab/resnet.ipynb
* https://colab.research.google.com/github/scotttodd/iree/blob/colab-refresh/colab/tensorflow_hub_import.ipynb
* https://colab.research.google.com/github/scotttodd/iree/blob/colab-refresh/iree/samples/variables_and_state/variables_and_state.ipynb